floss/language/go/coverage.py

```diff
@@ -79,8 +79,7 @@ def get_extract_stats(pe, all_ss_strings: List[StaticString], go_strings, min_le
         if secname != ".rdata":
             continue
 
-        if len(s.string) <= 30:
-            # guessed value right now
+        if len(s.string) <= 2800:
```

Comment:
you should add a comment and reference to your research here on why this value was chosen